examples/custom_plot_manipulation/Cargo.toml

```
@@ -0,0 +1,12 @@
+[package]
+name = "custom_plot_manipulation"
+version = "0.1.0"
+authors = ["Ygor Souza <ygor.souza@protonmail.com>"]
+license = "MIT OR Apache-2.0"
+edition = "2021"
+rust-version = "1.65"
+publish = false
+
+
+[dependencies]
+eframe = { path = "../../crates/eframe" }
```

Please follow the structure of the other examples

```suggestion
eframe = { path = "../../crates/eframe", features = [
    "__screenshot", # __screenshot is so we can dump a screenshot using EFRAME_SCREENSHOT_TO
] }
```